alacritty/src/window_context.rs

```diff
@@ -391,7 +392,11 @@ impl WindowContext {
 
         // Request immediate re-draw if visual bell animation is not finished yet.
         if !self.display.visual_bell.completed() {
-            self.display.window.request_redraw();
+            let window_id = Some(self.display.window.id());
+            // Winit on Windows discards `redraw_requested` calls from the `RedrawRequested` event,
+            // thus use a `Wakeup` which is delivered reliably and also sets the `dirty` flag.
```

Comment:
I hope this is a bit more clear. I've updated winit docs to say that it doesn't work though. And yes, it's a Windows behavior, which will take time to fix in winit, since it should use something else to schedule redraws.

The bug is not new, it's just new for us, since we now draw on `RedrawRequsted`.